Dockerfiles/agent/windows/amd64/Dockerfile

```diff
@@ -10,6 +16,11 @@ USER ContainerAdministrator
 
 SHELL ["pwsh", "-Command", "$ErrorActionPreference = 'Stop';"]
 
+RUN if (-not (Test-Path /Windows/System32/netapi32.dll)) { \
+        Copy-Item /netapi32.dll -Destination /Windows/System32/netapi32.dll \

```

Comment:
You can use `Move-Item`

Reply:
Good point.

Note that this doesn’t remove the need for the `if` as both are failing to overwrite an existing file:
```
Step 7/7 : RUN Move-Item /netapi32.dll -Destination /Windows/System32/netapi32.dll -Force
 ---> Running in 62d4374fc7ec
Move-Item: Cannot create a file when that file already exists.
```